recipes/gliner/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "gliner" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gliner-{{ version }}.tar.gz
+  sha256: 2c00570066a016d3ec682fc0d1aa0c61e7318873343662233c6994ac37fb2a10
+
+build:
+  noarch: python  # won't work on windows due to missing pytorch win build
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools >=61.0.0
+    - pip
+  run:
+    - python >=3.8
+    - pytorch >=2.0.0
+    - transformers >=4.38.2
+    - huggingface_hub >=0.21.4
+    - python-flair >=0.13.1
+    - scipy ==1.12
+    - seqeval
+    - tqdm
+    - urllib3 <2.0.0,>=1.0.0  # inherited from flair
```

Comment:
What do you mean "inherited from flair"? Is it an extra dependency of flair that this package requires?

Reply:
yes, flair seems to require this pin as it won't build without it. Could also just be that grayskull didn't pick this up in which case we can just remove the comment.

Reply:
You can leave it there. I was just trying to understand what was going on.